pkgs/servers/foundationdb/default.nix

```diff
@@ -69,6 +69,7 @@ in with builtins; {
 
     patches = [
       ./patches/ldflags-6.0.patch
+      ./patches/misc-fixes-6.0.patch
```

Comment:
That name is not describing.

Reply:
Sure. I will revise it. Thank you for reviewing!

Reply:
Renamed to `include-fixes-6.0.patch` and added some description into the commit message. Is it good enough?